meta/parse.pl

```diff
@@ -1420,6 +1521,103 @@ sub CreateMetadataHeaderAndSource
 
     WriteSource "-1";
     WriteSource "};";
+sub ProcessStatUnit
+{
+    my ($stat, $unit) = @_;
+    if (not defined $unit)
+    {
+        return "OTAI_STAT_VALUE_UNIT_NORMAL";
+    }
+    return "OTAI_STAT_VALUE_UNIT_DBM" if $unit eq "dBm";
+    return "OTAI_STAT_VALUE_UNIT_DB" if $unit eq "dB";
+
+    return "OTAI_STAT_VALUE_UNIT_NORMAL";
+}
```

Comment:
instead of dealing with this, all values should be uint64_t and each stat itself should have hardcoded unit in which value is presented as we have in SAI now